spec/lib/api/api_config_spec.rb

```diff
@@ -17,9 +17,19 @@
         identifiers
       end
 
+      def add_identifier_to_set(identifier, set)
+        if identifier
+          if identifier.kind_of?(Array)
+            identifier.each {|x| set.add(x)}
+          else
+            set.add(identifier)
+          end
+        end
+      end
+
       let(:api_feature_identifiers) do
         collection_settings.each_with_object(Set.new) do |(_, cfg), set|
-          set.add(cfg[:identifier]) if cfg[:identifier]
+          add_identifier_to_set(cfg[:identifier], set)
```

Comment:
Can't this line just be changed to:

```
Array(cfg[:identifier]).each { |id| set.add(id) }
```

Without introducing the add_identifier_to_set method.

Reply:
yes, sure, it is much better.